Tests/TuistGeneratorAcceptanceTests/GenerateAcceptanceTests.swift

```diff
@@ -880,6 +880,33 @@ final class GenerateAcceptanceTestAppWithDefaultConfiguration: TuistAcceptanceTe
     }
 }
 
+final class GenerateAcceptanceTestsAppWithMetalOptions: TuistAcceptanceTestCase {
+    func test_app_with_metal_options() async throws {
+        try setUpFixture(.appWithMetalOptions)
+        try await run(GenerateCommand.self)
+
+        let xcodeprojPath = fixturePath.appending(components: ["App.xcodeproj"])
```

Comment:
```suggestion
```

`xcodeprojPath` is a variable of `TuistAcceptanceTestCase` and unless `tuist generate` in the fixture produces multiple `.xcodeproj` files in the root, it should be set and equal to `App.xcodeproj`.